db/tables.go

```diff
@@ -39,10 +57,53 @@ func (t *Table) ApplyPage(orderColumn string) {
 	t.QBuilder = t.QBuilder.OrderBy(orderColumn)
 }
 
-// DEPRECATED! Do not use this method for counting,
+func (t *Table) CountQuery() (string, []interface{}, error) {
+	rawSQL, args, err := t.QBuilder.ToSql()
+	if err != nil {
+		return "", nil, err
+	}
+
+	countQuery := strings.Replace(rawSQL, t.Columns, "count(*) as count", 1)
```

Comment:
1. Eugene recommends using `count (1)`
2. I think we should drop the limit and offeset:

```go
	rawSQL, args, err := t.QBuilder.RemoveLimit().RemoveOffset().ToSql()

```